infer/src/pulse/PulseFormula.ml

```diff
@@ -2878,6 +2953,8 @@ module Formula = struct
               term_eqs_v
               (phi.term_eqs, phi.term_eqs_occurrences)
       in
+      let raw_atom_conds = phi.raw_atom_conds in
+      let raw_term_conds = phi.raw_term_conds in
```

Comment:
might want to add a "TODO: scramble raw atoms and terms" here, or just set them to empty sets for now (this is used by the join so empty set = forgetting all the information = top?)